src/utils/electron-enhanced-commands.ts

```
@@ -373,10 +380,400 @@ export async function sendCommandToElectron(
         `;
         break;
 
+      case 'hover':
+        // Hover over an element
+        const hoverSelector = args?.selector || '';
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)